ext/sodium/config.m4

```diff
@@ -0,0 +1,51 @@
+dnl $Id$
+dnl config.m4 for extension libsodium
+
+PHP_ARG_WITH(libsodium, for libsodium support,
+[  --with-libsodium[[=DIR]]  Include libsodium support])
+
+if test "$PHP_LIBSODIUM" != "no"; then
+  SEARCH_PATH="/usr/local /usr"     # you might want to change this
+  SEARCH_FOR="/include/sodium.h"  # you most likely want to change this
+  if test -r $PHP_LIBSODIUM/$SEARCH_FOR; then # path given as parameter
+    LIBSODIUM_DIR=$PHP_LIBSODIUM
+  else # search default path list
+    AC_MSG_CHECKING([for libsodium files in default path])
+    for i in $SEARCH_PATH ; do
+      if test -r $i/$SEARCH_FOR; then
+        LIBSODIUM_DIR=$i
+        AC_MSG_RESULT(found in $i)
+      fi
+    done
+  fi
```

Comment:
See some other ext for example (zip, interbase, ...)
Else I will take care of it.